addons/mail/static/src/core/common/store_service.js

```diff
@@ -418,14 +418,14 @@ export class Store extends BaseStore {
                 message_type: "comment",
                 partner_ids,
                 subtype_xmlid: subtype,
+                special_mentions: validMentions?.specialMentions ?? [],
```

Comment:
It would be better to not send the key at all if it's empty.

Also if `@everyone` has no effect on chatter we should not be able to select it.

Reply:
This should be a fix in earlier version if the bug can be reproduced.

The change in python is just more defensive code and cleanup, ok for master.

Reply:
But there is still a crash if we type it?

Reply:
Yes